### PR TITLE
Add side panels for commit pipeline visualization

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## Fixes 
Fixes [HAC-2578](https://issues.redhat.com/browse/HAC-2578)

## Description
Add support for node click in the commit visualization. Upon a click a side panel is shown withe the appropriate information for the selected node.

## Type of change
- [x] Feature

### Commit:
image

### Build:
image

### Integration Test:
image

### Environment:
image

### Release:
image

### Managed Environment:
image

/cc @MariaLeonova @karthikjeeyar @christianvogt

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
Need to add tests still

**Comment by MariaLeonova** [comment]
@jeff-phillips-18 Vulnerability scan results UI has been updated by @christianvogt in [this PR](https://github.com/openshift/hac-dev/pull/517#issuecomment-1483190035) and should be the same here.
 
Screenshot 2023-03-28 at 13 53 28

**Comment by karthikjeeyar** [comment]
Sidebar heading for component test empty state is should not be a link (right now it takes you to 404 page).

image

**Comment by karthikjeeyar** [comment]
@MariaLeonova We are not showing the release and managed environment nodes in lifecycle visualization today, we have hidden this under the MVP feature flag. If we need to show this in commits page, then we must show in lifecycle graph too or else it would be confusing to the user.  Do you know if we are bringing back in lifecycle graph? or shall we hide this release/managed env until we show in lifecycle graph? 


image

**Comment by jeff-phillips-18** [comment]
> @MariaLeonova We are not showing the release and managed environment nodes in lifecycle visualization today, we have hidden this under the MVP feature flag. If we need to show this in commits page, then we must show in lifecycle graph too or else it would be confusing to the user. Do you know if we are bringing back in lifecycle graph? or shall we hide this release/managed env until we show in lifecycle graph?

It is hidden behind the MVP flag here as well, I just turned it on to show in the screenshot.

File: src/components/Commits/CommitDetails/visualization/useCommitWorkflowData.ts [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+          ];
+
+      nodes.push(...staticEnvNodes);
+      const staticEnvNodesWidth = staticEnvNodes.reduce(
+        (max, node) => Math.max(max, node.width),
+        0,
+      );
+      staticEnvNodes.forEach((n) => (n.width = staticEnvNodesWidth));
+
+      if (mvpFeature) {
```
Comment (karthikjeeyar) [inline src/components/Commits/CommitDetails/visualization/useCommitWorkflowData.ts:339]:
```suggestion
      if (!mvpFeature) {
```

**Pull request merged**